views/index.html.twig

```diff
@@ -81,3 +81,20 @@
         </div>
     </div>
 {% endblock %}
+
+{% block pagejs %}
+<script>
+$(document).ready(function () {
+  // Add badge for count of Deceased Participant Reports for review
+  var deceasedReportCount = 0;
+  $.ajax({
+    url: '/s/deceased-reports/stats'
+  }).done(function (data) {
+    deceasedReportCount = data.pending
+    if (deceasedReportCount > 0) {
+      $('#deceased_report_block').prepend($('<div class="welcome-page-badge">' + deceasedReportCount +'</div>'));
```

Comment:
This causes a problem with the `equalizeColumnHeight` plugin in `app.js`. The purpose of this plugin is to make all of the contents across columns the same height. The plugin assumes that the immediate children of the column divs should be equalized (the btn-block elements in this case). Is it possible to make this work where the badge is inside the `a.btn-block` instead of the parent column div?

Here's what currently happens when you resize the window after the badge is loaded:

![Screen Shot 2020-09-14 at 3 18 59 PM](https://user-images.githubusercontent.com/860499/93133969-a5316e80-f69d-11ea-9e08-0f67c62c790a.png)

Reply:
Resolved in ed8c73e. Has the added bonus of the badge being clickable as well. Needed to adjust the font size down a smidge.